db/migrate/20170206211813_add_references_to_user_bet.rb

```diff
@@ -0,0 +1,12 @@
+class AddReferencesToUserBet < ActiveRecord::Migration[5.0]
+  def change
+    change_table :user_bets do |t|
```

Comment:
This is also bad. 

What effect would this have if it were run on a production database? 
What is the goal of the migration? (what are you trying to accomplish) 
Can you achieve that goal in a less destructive way?